### PR TITLE
Support for Jels60

### PR DESCRIPTION
**Pull request by Jels02**
Add Jels60

## Description

Add support for Jels60 PCB

## Types of Changes

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jels/jels60/rules.mk [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# MCU name
+MCU = atmega32u4
+# Bootloader
```
Comment (fauxpark) [inline keyboards/jels/jels60/rules.mk:3]:
```suggestion

# Bootloader selection
```

File: keyboards/jels/jels60/rules.mk [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic
```
Comment (fauxpark) [inline keyboards/jels/jels60/rules.mk:9]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/jels/jels60/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#include QMK_KEYBOARD_H
+
+#define ________ _______
+
```
Comment (fauxpark) [inline keyboards/jels/jels60/keymaps/via/keymap.c:4]:
```suggestion
```

File: keyboards/jels/jels60/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#include QMK_KEYBOARD_H
+
+#define ________ _______
+
```
Comment (fauxpark) [inline keyboards/jels/jels60/keymaps/default/keymap.c:4]:
```suggestion
```

File: keyboards/jels/jels60/jels60.c [hunk 5]
```diff
@@ -0,0 +1,7 @@
+// shhh... nothing to see here ;)
+
+#include "jels60.h"
+
+
+
+
```
Comment (fauxpark) [inline keyboards/jels/jels60/jels60.c:7]:
```suggestion
#include "jels60.h"
```

File: keyboards/jels/jels60/config.h [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor paramater */
+#define VENDOR_ID       0x006a    //Jels (J)
```
Comment (fauxpark) [inline keyboards/jels/jels60/config.h:22]:
```suggestion
#define VENDOR_ID       0x006A    //Jels (J)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/jels/jels60/jels60.c [hunk 7]
```diff
@@ -0,0 +1,5 @@
+#include "jels60.h"
```
Comment (drashna) [inline keyboards/jels/jels60/jels60.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/jels/jels60/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/jels/jels60/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/jels/jels60/keymaps/via/keymap.c [hunk 9]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/jels/jels60/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/jels/jels60/config.h [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
+
+
+
```
Comment (drashna) [inline keyboards/jels/jels60/config.h:54]: These are the defaults, so don't really need to be defined
```suggestion
```

File: keyboards/jels/jels60/readme.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+# Jels60
+
+![jels60](https://i.imgur.com/3Ec4YZN.png)
```
Comment (drashna) [inline keyboards/jels/jels60/readme.md:3]:
```suggestion
![jels60](https://i.imgur.com/3Ec4YZNh.png)
```

**Review by drashna — APPROVED** [review]

**Pull request merged**